viewer/src/datamodels/pointcloud/types.ts

```diff
@@ -42,3 +42,82 @@ export enum PotreePointSizeType {
   Adaptive = Potree.PointSizeType.ADAPTIVE,
   Fixed = Potree.PointSizeType.FIXED
 }
+
+
+/**
+ * ASPRS well known point class types.
+ * @see {@link http://www.asprs.org/wp-content/uploads/2019/03/LAS_1_4_r14.pdf} (page 30)
+ */
+export enum WellKnownPointClassTypes {
```

Comment:
Does it relative to Potree.PointColorType.CLASSIFICATION? Should it extend it or something?

Reply:
`Potree.PointColorType.CLASSIFICATION` determines how to color the model, this enum is used to control what points are visible.

Reply:
Perhaps I just posted the wrong enum, but doesn't potree have something related to it? If not OK – nothing to discuss here. 

Also, actually, enum extension is not something supported in typescript, so...